README.md

```diff
@@ -1,62 +1,67 @@
 
-# Studio
+# SciLifeLab Serve
```

Comment:
Perhaps we can add github action badges aswell?
![example](https://github.com/ScilifelabDataCentre/stackn/actions/workflows/build-and-test-studio.yaml/badge.svg?branch=develop)

Reply:
We should change the workflow name to serve instead of studio